pkg/cable/wireguard/driver.go

```diff
@@ -58,15 +61,15 @@ type wireguard struct {
 }
 
 // NewDriver creates a new WireGuard driver
-func NewDriver(localSubnets []string, localEndpoint types.SubmarinerEndpoint) (cable.Driver, error) {
+func NewDriver(subnets []string, endpoint types.SubmarinerEndpoint, cluster types.SubmarinerCluster) (cable.Driver, error) {
```

Comment:
localSubnets/localEndpoint?
```suggestion
func NewDriver(localSubnets []string, localEndpoint types.SubmarinerEndpoint, cluster types.SubmarinerCluster) (cable.Driver, error) {
```

Reply:
DONE